web/beacon-app/src/features/teams/components/DeleteMember/DeleteMemberModal.tsx

```diff
@@ -0,0 +1,78 @@
+import { Modal } from '@rotational/beacon-core';
+import { useEffect } from 'react';
+import { toast } from 'react-hot-toast';
+
+import { Close } from '@/components/icons/close';
+import Button from '@/components/ui/Button/Button';
+import { useDeleteMember } from '@/features/members/hooks/useDeleteMember';
+
+import DeleteMemberForm from './DeleteMemberForm';
+
+type DeleteMemberModalProps = {
+  onOpen: {
+    opened: boolean;
+    member?: any;
+  };
+  onClose: () => void;
+};
+
+function DeleteMemberModal({ onClose, onOpen }: DeleteMemberModalProps) {
+  const { opened, member } = onOpen;
+  const { deleteMember, isDeletingMember, wasMemberDeleted, hasMemberFailed, error, reset } =
+    useDeleteMember(member?.id);
+  const initialValues = {
+    id: member?.id,
+    name: member?.name || member?.email || '---',
```

Comment:
I believe the table currently displays `-` in the `name` field for invited users. Should it be updated to `---` to match the initial value?